### ISSUE TITLE
^M error when PluginSearch ,Please help me~

### ISSUE THREAD

**Issue by THtianhao** [VundleVim/Vundle.vim#914]
Do you have this question? When i input PluginSearch xx and Tab ,I will occur some error like this:
```
:PluginSearch ...
Error detected while processing function vundle#scripts#complete[6]..<SNR>58_load_scripts:
line    7:
E15: Invalid expression: <html>^M
E15: Invalid expression: <html>^M
Error detected while processing function vundle#scripts#complete:
line    6:
E714: List required
:PluginSearch 0
```

**Comment by davidlowryduda** [comment]
Yes, I confirm this behaviour. This is caused by the fact that the vim-scripts mirror api has been deprecated. (See https://vim-scraper.github.io/). As a result, the stashed json file of available vim scripts is just a HTML file containing info that the resource has moved, and the parser fails.

The appropriate thing to do is to remove this functionality from vundle.

**Comment by michaelkruglos** [comment]
@davidlowryduda I strongly disagree. There's another api which can be implemented. 
`https://vimawesome.com/api/plugins?query=`
Example request with fetch on node:
```javascript
var fetch = require('node-fetch')

fetch("https://vimawesome.com/api/plugins?query=golang&page=1", {
  "credentials":"omit",
  "headers":{
    "accept":"application/json, text/javascript, */*; q=0.01",
    "accept-language":"en-US,en;q=0.9",
    "sec-fetch-mode":"cors",
    "sec-fetch-site":"same-origin",
    "x-requested-with":"XMLHttpRequest"
  },
  "referrer":"https://vimawesome.com/",
  "referrerPolicy":"no-referrer-when-downgrade",
  "body":null,
  "method":"GET",
  "mode":"cors"
})
  .then(result=>result.json())
  .then(json=>console.log(json)) ;
```

**Comment by erasmussen1** [comment]
> Yes, I confirm this behaviour. This is caused by the fact that the vim-scripts mirror api has been deprecated. (See https://vim-scraper.github.io/). As a result, the stashed json file of available vim scripts is just a HTML file containing info that the resource has moved, and the parser fails.
> 
> The appropriate thing to do is to remove this functionality from vundle.



> Yes, I confirm this behaviour. This is caused by the fact that the vim-scripts mirror api has been deprecated. (See https://vim-scraper.github.io/). As a result, the stashed json file of available vim scripts is just a HTML file containing info that the resource has moved, and the parser fails.
> 
> The appropriate thing to do is to remove this functionality from vundle.

Is there a workaround this issue? (I'm not a plugin expert)

**Comment by davidlowryduda** [comment]
@erasmussen1 It's not quite clear what you mean by workaround. If you don't use `PluginSearch`, then vundle works fine. I might call "google for a plugin you're interested in" a viable workaround. Did you have another meaning in mind?

**Comment by ShriprajwalK** [comment]
So do we have to manually add Plugins in .vimrc and then use `PluginInstall` from now on?

**Comment by AntonVLA** [comment]
I have same problem

**Comment by Rimvel-hub** [comment]
I also have this problem. Is there a solution?

**Comment by San4ito** [comment]
Me too, and i don't know how to solve this problem.

**Comment by r0mille** [comment]
plus 1

**Comment by kmuzalewska** [comment]
problem still present

**Comment by nhannguyen1295** [comment]
Now problem still present and cant fix it
![image](https://user-images.githubusercontent.com/59357709/101895881-b3c6ed80-3bda-11eb-9ed5-20a02ffef36b.png)

**Comment by nhannguyen1295** [comment]
Now problem still present, can't fix it.
![image](https://user-images.githubusercontent.com/59357709/101895964-d5c07000-3bda-11eb-86d3-f39bcd210fb5.png)

**Comment by Unixware** [comment]
its the f year 2021 still this issue.. for LOLz

**Comment by huberemanuel** [comment]
Same here

**Comment by mrfhitz** [comment]
i have the same problem, looks like its not gonna be solved

**Comment by antoniopaolini** [comment]
Maybe [this ](https://github.com/VundleVim/Vundle.vim/issues/6 )solve the problem.
Also see [this](https://github.com/vim/vim/issues/6156)
